corehq/apps/enterprise/models.py

```diff
@@ -11,7 +11,7 @@ class EnterprisePermissions(models.Model):
     "source" domain to be automatically granted their same set of permissions in all domains
     controlled by the config.
     """
-    account = models.ForeignKey('accounting.BillingAccount', null=False, on_delete=models.CASCADE)
+    account = models.ForeignKey('accounting.BillingAccount', null=False, on_delete=models.CASCADE, unique=True)
```

Comment:
I think this introduced a django check warning, which appears if you run `manage.py check` or some other common management commands.
```
(cchq) ➜  commcare-hq git:(master) ✗ ./manage.py check
...
System check identified some issues:

WARNINGS:
enterprise.EnterprisePermissions.account: (fields.W342) Setting unique=True on a ForeignKey has the same effect as using a OneToOneField.
	HINT: ForeignKey(unique=True) is usually better served by a OneToOneField.
```
@orangejenny